RELEASES.md

```diff
@@ -28,12 +28,19 @@ Decide among the maintainers if a new release is due.
 Dear all,
 
 Today, $NEW_VERSION of the openstreetmap-carto stylesheet (the default
-stylesheet on openstreetmap.org) has been released.
-
-If the tile server admins have rolled out the new version, add
-
-  and rolled out to the openstreetmap.org servers. It might take a
-  couple of days before all tiles show the new rendering.
+stylesheet on openstreetmap.org) has been released. The actual deployment 
+on OSM website is done by sysadmin team, so there are no guarantees 
+if and when the changes will be visible there.
+
+If the tile server admins have rolled out the new version, change it to:
+
+  Today, $NEW_VERSION of the openstreetmap-carto stylesheet (the default
+  stylesheet on openstreetmap.org) has been released and rolled out 
+  to the openstreetmap.org servers. It might take a couple of days 
+  before all tiles show the new rendering. 
+  
+While developing standard OSM map is our primary goal, there are also 
```

Comment:
This reads like we're elevating tile.osm.org above others

Reply:
I wanted to show how we design in reality (I'm not even aware where other direct deployments are), but if you like it more neutral, it can be for example:

`This style is not only used for rendering standard tile layer on OSM.org, it can be 
also deployed in other, independent websites and forks.`

Reply:
Or:

`This style can be also deployed in other websites and forks on their own schedule.`

Reply:
"This reads like we're elevating tile.osm.org above others" - I do not think that it is a problem.